server/tests/message.integration.spec.js

```diff
@@ -330,6 +341,60 @@ describe('Message API:', () => {
             })
         );
 
+        it('has an option to filter by sent messages', () => request(app)
+            .get(`${baseURL}/message/list?sent=true`)
+            .set('Authorization', `Bearer ${auth}`)
+            .expect(httpStatus.OK)
+            .then((res) => {
+                expect(res.body).to.be.an('array');
```

Comment:
test for a non-zero length here, this is not catching errors because there are no array entries to run the loop on